app/views/advanced_messenger/_html_head.html.erb

```diff
@@ -1,15 +1,227 @@
+                } else {
+                    notesPreviewDiv.html("<span class='collapse-message-read'><%=t(:message_collapsed_read)%></span><span class='collapse-message-preview'>" + notesPreview + "...</span>");
+                }
+            } else {
+                notesPreviewDiv.html("<span class='collapse-message'><%=t(:message_collapsed)%></span><span class='collapse-message-preview'>" + notesPreview + "...</span>");
+            }
+        }
+
+        function renderReadIndicatorsForAllUsers(readableEntityId, currentNode, currentUserId, creationDate, users, readStatuses) {
+            let userInitials;
+            let color;
+            let tooltipUserList = "";
+            let usersColoredInitials = "<span style='padding-right: 5px'>";
+            let isSomeoneInterestedIn= Object.keys(users).length > 0;
+            let i = 0;
+            for (const [userId, user] of Object.entries(users)) {
+                const readStatus = readStatuses[userId] 
+                userInitials = user.firstname.charAt(0).toUpperCase() + user.lastname.charAt(0).toUpperCase(); 
+                colorStyle = "style='color:" + (readStatus["read"] == 0 ? RED : (readStatus["read"] == 3 ? GREY : GREEN)) + "'";
+                tooltipUserList += "<li " + colorStyle.replace(GREEN, "green") + ">" + userInitials + " - " + user.link +
+                                        " (" + (readStatus["read"] == 0 ? "<%=t(:message_notifications_tooltip_list_item_not_viewed)%>" : readStatus["date"]) + ")" + 
+                                    "</li>";
+                usersColoredInitials +="<span " + colorStyle + ">" + userInitials + "</span>";
+                if (i < Object.keys(users).length - 1) {
+                    usersColoredInitials += ", ";
+                }
+                i++;
+            }
+            usersColoredInitials += "</span>";
+
+            const isOfInterestForCurrentUser = false;
+            for (const user in users) {
+                if (user.id == currentUserId) {
+                    isOfInterestForCurrentUser = true;
+                    break;
+                }
+            }
+            if (isSomeoneInterestedIn) {
+                const readUnreadIndicators = currentNode.find('.users_initials_and_tooltip_placeholder');
+                let tooltipTriggerId = "others-notifications-tooltip-" + readableEntityId;
+                const tooltip = "<a id='" + tooltipTriggerId + "' class='icon icon-chat' title=' ' style='cursor: pointer'></a>";
+                tooltipTriggerId = "#" + tooltipTriggerId;   
+
+                readUnreadIndicators.html((isOfInterestForCurrentUser ? " | " : "") + usersColoredInitials + tooltip);
+                
+                // ====== tooltip ======
+                // the `ready` function was needed because otherwise the `show` / `hide` effect didn't had any effect 
+                $(document).ready(function() {
```

Comment:
If you have an external link describing the problem (like stackoverflow, ...) could you added in the comment to this block?

Reply:
I didn't found any resource about this exact problem, but I found a slightly different problem related to jquery ui tooltip not displaying. One of the answers here worked for our problem also: 
// https://stackoverflow.com/questions/15534061/jquery-tooltip-not-displaying

I modified the comment in code to include this remark